### PR TITLE
redesigned cancel page to new design

### PR DESCRIPTION
**Pull request by PeerRich**
before: 

CleanShot 2022-08-22 at 12 50 59@2x

after:

CleanShot 2022-08-22 at 12 50 40@2x

### CODE REVIEW

**Comment by PeerRich** [comment]
thank you @Udit-takkar for some of the changes in a previous PR

File: apps/web/pages/cancel/[uid].tsx [hunk 1]
```diff
@@ -43,24 +43,17 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
-              <div
-                className="inline-block transform overflow-hidden rounded-lg bg-white px-4 pt-5 pb-4 text-left align-bottom shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-sm sm:p-6 sm:align-middle"
-                role="dialog"
-                aria-modal="true"
-                aria-labelledby="modal-headline">
+      <main className="h-full sm:flex sm:items-center">
+        <div className="mx-auto flex justify-center px-4 pt-4 pb-20 sm:block sm:p-0">
+          <div className="inline-block transform overflow-hidden rounded-md border bg-white px-8 pt-5  pb-4 text-left align-bottom transition-all dark:border-neutral-700 dark:bg-gray-800 sm:my-8 sm:w-full sm:max-w-lg sm:py-6 sm:align-middle">
+            <div className="">
+              <div className="">
```
Comment (emrysal) [inline apps/web/pages/cancel/[uid].tsx:56]: ?.?

Comment (PeerRich) [inline apps/web/pages/cancel/[uid].tsx:56]:
```suggestion
            <div>
              <div>
```

File: apps/web/pages/cancel/[uid].tsx [hunk 2]
```diff
@@ -98,82 +89,83 @@ export default function Type(props: inferSSRProps<typeof getServerSideProps>) {
...
-                                    </div>
+                        <div className="border-bookinglightest text-bookingdark mt-4 grid grid-cols-3 border-t py-4 text-left dark:border-gray-900 dark:text-gray-300">
+                          <div className="font-medium">{t("what")}</div>
+                          <div className="col-span-2 mb-6">{props.booking?.title}</div>
+                          <div className="font-medium">{t("when")}</div>
+                          <div className="col-span-2 mb-6">
+                            {props.booking?.eventType.recurringEvent && props.recurringInstances ? (
+                              <>
+                                <div className="mb-1 inline py-1 text-left">
+                                  <div className="">
```
Comment (emrysal) [inline apps/web/pages/cancel/[uid].tsx:100]:
```suggestion
                                  <div>
```

**Review by emrysal — APPROVED** [review]

**Review by zomars — APPROVED** [review]

**Pull request merged**